Gems/Atom/RHI/Vulkan/Code/Source/RHI/FrameGraphExecuteGroupSecondaryHandler.cpp

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#include <RHI/CommandList.h>
+#include <RHI/Device.h>
+#include <RHI/FrameGraphExecuteGroupSecondaryHandler.h>
+#include <RHI/FrameGraphExecuteGroupSecondary.h>
+#include <RHI/RenderPass.h>
+#include <RHI/RenderPassBuilder.h>
+
+namespace AZ
+{
+    namespace Vulkan
```

Comment:
nbd this might be an opportunity to reduce the indent level of these moved files with `namespace AZ::Vulkan {`